docusaurus.config.js

```diff
@@ -1,73 +1,74 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-import { themes as prismThemes } from 'prism-react-renderer';
+import { themes as prismThemes } from "prism-react-renderer";
```

Comment:
I though you wanted to keep the formatting?

Reply:
Fixed